guides/micronaut-turbo-native/src/main/resources/views/turbo-streams.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
+<html lang="en" th:replace="~{layoutFile :: layout(~{::section})}" xmlns:th="http://www.thymeleaf.org">
+<section class="grid">
+    <h1 class="page-title display-native --turbo-streams">
+        <div class="actions__icon --turbo-streams" aria-hidden="true"></div>
+        Turbo Streams: Deliver live page changes
+    </h1>
+
+    <p>
+        Making partial page changes in response to asynchronous actions is how we make the application feel alive. While Turbo Frames give us such updates in response to direct interactions within a single frame, Turbo Streams let us change any part of the page in response to updates sent over a WebSocket connection, SSE or other transport. (Think an imbox that automatically updates when a new email arrives.)
```

Comment:
```suggestion
        Making partial page changes in response to asynchronous actions is how we make the application feel alive. While Turbo Frames give us such updates in response to direct interactions within a single frame, Turbo Streams let us change any part of the page in response to updates sent over a WebSocket connection, SSE, or other transport. (Think an inbox that automatically updates when a new email arrives.)
```

Reply:
imbox is not a typo. it is a feature of Hey